index.js

```diff
@@ -6,12 +6,30 @@ var AbstractIterator = require('abstract-leveldown').AbstractIterator
 var inherits = require('inherits')
 var Codec = require('level-codec')
 var EncodingError = require('level-errors').EncodingError
+var rangeMethods = ['approximateSize', 'compactRange']
 
 module.exports = DB.default = DB
 
 function DB (db, opts) {
   if (!(this instanceof DB)) return new DB(db, opts)
-  AbstractLevelDOWN.call(this, '')
+
+  AbstractLevelDOWN.call(this, db.supports)
```

Comment:
Bug: this takes over `additionalMethods` of the underlying db, but we don't proxy those methods. Our `additionalMethods` should (initially) be empty.